src/content/configuration/stats.md

```diff
@@ -316,6 +316,21 @@ module.exports = {
 };
 ```
 
+### `stats.orphanModules`
+
+`boolean = false`
+
+Tells `stats` whether to hide `orphan` modules. A module is an `orphan` if it is not included in any chunk. Orphans modules are now hidden by default in `Stats`.
```

Comment:
When we say **now**, what exactly do we mean? suggest we remove it, will throw a pr. Also this paragraph uses `stats` and capital first `Stats`. Gona fix it along the way